app/controllers/application_controller.rb

```diff
@@ -111,6 +111,7 @@ def use_collection(id)
   def clear_session
     store_oauth_token()
     session[:user_id] = nil
+    session[:user_name] = nil
```

Comment:
For these sorts of generally useful comments, could you please put them in code comments instead of pull request comments so we can understand the code better in the future?